repository_service_tuf/helpers/tuf.py

```diff
@@ -200,32 +204,39 @@ def _update_snapshot(
     # Update top-level 'targets' role, to delegate trust for all target files
     # to 'bins' role, defining target path patterns, trusted signing keys and
     # required signature thresholds.
-    targets = _load(Targets.type)
-    targets.signed.delegations = Delegations(keys={}, roles={})
-    targets.signed.delegations.roles[BIN] = DelegatedRole(
-        name=BIN,
-        keyids=[],
-        threshold=settings[BIN].threshold,
-        terminating=False,
-        paths=settings[Targets.type].paths,
-    )
+    targets_metadata = _load(Targets.type)
+    targets_metadata.signed.delegations = Delegations(keys={}, roles={})
+    targets_metadata_roles: dict[str, DelegatedRole] = {
+        BIN: DelegatedRole(
+            name=BIN,
+            keyids=[],
+            threshold=settings[BIN].threshold,
+            terminating=False,
+            paths=settings[Targets.type].paths,
+        )
+    }
+
+    assert targets_metadata_roles is not None
+
+    targets_metadata.signed.delegations.roles = targets_metadata_roles
```

Comment:
You will have to update this part when you rebase, sorry about that.
We have removed the `bins` role as a whole and now `targets` is using `succinct hash bin delegation`:
https://github.com/vmware/repository-service-tuf-cli/pull/175